modules/test/playwright/tests/object-web/objectFolder.spec.ts

```diff
@@ -91,6 +91,21 @@ test.describe('manage object definitions through model builder', () => {
 
 		await apiHelpers.objectAdmin.deleteObjectFolder(objectFolder.id);
 	});
+
+	test('can navigate from Model Builder to Account Settings', async ({
+		modelBuilderPage,
+		page,
+	}) => {
+		await modelBuilderPage.goto({objectFolderName: 'Default'});
+
+		await modelBuilderPage.toggleSidebarsButton.click();
+
+		await page.locator('button[data-qa-id="userPersonalMenu"]').click();
+
+		await page.getByRole('menuitem', {name: 'Account Settings'}).click();
```

Comment:
Hey @igor-franca, We have an `goToAccountSettings()` method in the AccountSettingsPage, containing these same actions, so you could have this line instead
`await accountSettingsPage.goToAccountSettings();`

However, the locators they are using are a little off and you would have to fix it as well, it's a minor change, I've tested locally and it works. But feel free to do as you see fit!

Reply:
Hi @larissacribeiro, I decided not to change the `userPersonalMenuButton` locator. I could use `getByTitle('User Profile Menu')`, but the testId approach can be useful when we want to create a localizer that cannot be affected by language changes. I think this is the real reason for this use, even if it doesn't work for our context. 

But I took the opportunity to use the other locators and create a new one on the Account Settings page, thanks!